### DISCUSSION TITLE
Replace the "Show less activity like this" button with a "hide content like this" button

### DISCUSSION THREAD

**Discussion by allo-** [community/community#113653] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Please replace the "Show less activity like this" button with a "Hide content like this" button.

There is no reason not to respect users' preferences to *completely* hide certain categories of items from the front page. GitHub is not a social network that needs to inject ads into people's feeds, so please give us the option to completely hide items on the front page that we do not want to see there.

"Show less like this" is a dark pattern that tries to circumvent users' wishes to hide content, when it would be easy to implement actual consent and add a real "Hide this category" button.

**Comment by XeroOl** [comment]
I have pushed "show less like this" every time I see one of these, and it no longer gives me the option, but it still hasn't removed the algorithmic recommendations

**Comment by Sisis700** [comment]
[](url)
